Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -131,6 +131,21 @@ public class JavaScriptLifter: Lifter {
                 return arrayPattern
             }
 
+            // Helper function to lift destruct object operations
+            func liftObjectPattern(properties: [String], outputs: [String], hasRestElement: Bool) -> String {
```

Comment:
Nit: maybe rename here and in FuzzILLifter to `liftObjectDestructPattern`? Then you can also drop the comment. If you want, you can also pass in the destructured object variable.

Reply:
I'd like to leave the function signature as is, the reason being that I want to reuse this when I create ops that lift a function param list as destructible object. Ex:
```js
function f({"prop1":v1, "prop2":v2}) {
  return v1 + v2;
}
```